package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uscreentv/v-color",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
It should be `1.3.0`, because it's not a fix, it's a minor change

Reply:
@lamartire done